github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/trigger/JobRunnerForCause.java

```diff
@@ -82,6 +87,42 @@ public boolean apply(GitHubPRCause cause) {
                 }
             }
 
+            if (trigger.isAbortRunning()) {
+                Computer[] computers = getJenkinsInstance().getComputers();
+                for (Computer computer : computers) {
+                    if (isNull(computer)) {
+                        continue;
+                    }
+
+                    List<Executor> executors = computer.getExecutors();
+
+                    for (Executor executor : executors) {
+                        if (isNull(executor)) {
+                            continue;
+                        }
+
+                        if (executor.isBusy()) {
+                            Queue.Executable executable = executor.getCurrentExecutable();
+
+                            Queue.Task task = executor.getCurrentWorkUnit().work.getOwnerTask();
+                            if (task instanceof AbstractProject<?, ?>) {
+                                AbstractBuild<?, ?> abstractBuild = (AbstractBuild<?, ?>) task;
+                                if (jobTask.getFullDisplayName().equals(job.getFullDisplayName())) {
+                                    jobTask.
+                                }
+                            } else if (task instanceof Job) {
+                                Job<?, ?> jobTask = (Job<?, ?>) task;
```

Comment:
idea was abort abstracbuild as it have no any complex runs like matrix or workflow. And then subcheck Jobs... but i completely confused.